subgraphs/orbit/src/mappings/helpers.ts

```diff
@@ -0,0 +1,91 @@
+// import { log } from "@graphprotocol/graph-ts"
+import { Address, ethereum } from "@graphprotocol/graph-ts";
+import { LiquidityPool, LiquidityPoolFee } from "../../generated/schema";
+import { Pair as PairTemplate } from "../../generated/templates";
+import { Factory as FactoryContract } from "../../generated/templates/Pair/Factory";
+import {
+  BIGDECIMAL_ONE,
+  BIGDECIMAL_TWO,
+  BIGDECIMAL_ZERO,
+  BIGINT_ZERO,
+  FACTORY_ADDRESS,
+  LiquidityPoolFeeType,
+  LP_FEE_TO_ON,
+  PROTOCOL_FEE_TO_ON,
+  TRADING_FEE,
+} from "../common/constants";
+import { getOrCreateDex, getOrCreateToken } from "../common/getters";
+
+export const factoryContract = FactoryContract.bind(
```

Comment:
let's also move this to common/constants.ts

Reply:
Deleted the entire file itself since it wasn't being used.